adoc/Release-Notes.adoc

```diff
@@ -21,6 +21,67 @@ endif::[]
 [id='sec.major-change']
 == Major Changes
 
+[id='sec.1_2_1']
+=== Release 1.2.1, September 2018
+
+[id='sec.1_2_1.issue']
+==== Known Issues
+
+* If you are upgrading to {product} 1.2.1, you will need to run the `helm upgrade` command with two additional parameters: `helm upgrade --force --recreate-pods`. This will be the case with all future 1.x upgrades due to the expected behaviour with Kubernetes that unready pods won’t get upgraded.
```

Comment:
Suggestion that puts the "issue" first, then discusses the "solution" which improves clarity and also use simpler tenses:
"Starting with {product} 1.2.1, during `helm upgrade`, Kubernetes will not upgrade pods that are not ready by default. To upgrade them anyway, use the following amended command instead: `helm upgrade --force --recreate-pods`."

I hope I have not misunderstood the content. Feel free to adjust.

Reply:
Makes total sense. I'll review the rest of your feedback in the "outdated" comments, thanks!

Reply:
Updated entry, though using some of the similar 1.2.0 copy.